code/datums/mutations/body.dm

```diff
@@ -147,16 +147,19 @@
 	ADD_TRAIT(owner, TRAIT_TOO_TALL, GENETIC_MUTATION)
 	owner.visible_message(span_danger("[owner] suddenly grows tall!"), span_notice("You feel a small strange urge to fight small men with slingshots. Or maybe play some basketball."))
 	RegisterSignal(owner, COMSIG_MOVABLE_MOVED, PROC_REF(head_bonk))
+	owner.apply_height_filters()
```

Comment:
This is doing nothing, either use `regenerate_icons` or do what dwarfism does and register a signaladd/signalremove to call `regenerate_icons` when `TRAIT_TOO_TALL` is added

Reply:
i already did that in local but now you suggested it before i pushed and it's gonna look like youre the one with the idea ,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.,.